### ISSUE TITLE
[SUPPORT] documentation for cspline fitting

### ISSUE THREAD

**Issue by barentine** [python-microscopy/python-microscopy#1186]
Would be great if `Analysing Localisation Microscopy data` section of the docsincluded how to save/load/use an experimental PSF

**Comment by David-Baddeley** [comment]
Agreed - I thought we did have something on this, but maybe not ...

**Comment by David-Baddeley** [comment]
Docs for saving here:

https://python-microscopy.org/doc/PSFExtraction.html

Really need to work out what is going wrong with formatting

**Comment by David-Baddeley** [comment]
I'd probably opt for a separate 3D fitting page which was linked from the base localisation page

**Comment by David-Baddeley** [comment]
Perhaps also a flowchart outlining how to choose localisation modes ...
